test.js

```diff
@@ -2,6 +2,10 @@ import test from 'ava';
 import getStream from 'get-stream';
 import fn from './';
 
+global.Promise = Promise;
```

Comment:
Necessary for Node 0.10. `get-stream` isn't using a Promise ponyfill and thus should be exposed in the global scope to be able to test on 0.10.